server/models/dashboard.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
```

Comment:
I like how you created separate model files to separate the schemas, more appropriate for the anticipated app size.